content/docs/06-integrations/00-cloudanix.md

```diff
@@ -70,12 +70,42 @@ This Helm Chart installs four Cloudanix services to enable container security ca
 - **config-cron**: A job that runs periodically in a Kubernetes cluster to maintain the configuration of Cloudanix inventory and threat services.
 - **misconfig-cron**: A job that captures Kubernetes misconfigurations and displays them on the Cloudanix dashboard.
 - **inventory-service**: An inventory service that detects any new Kubernetes resources and displays them on the Cloudanix dashboard.
-- **threat-service**: A threat service that exports threat events and affected Kubernetes resources which are visible on the Cloudanix dashboard.
+- **threat-service**: A threat service that exports threat events and affected Kubernetes resources, which are visible on the Cloudanix dashboard.
 
 
 
 From the **Workloads** page, click the **Risks** tab to view a list of failed threat rules. You can exclude resources, such as pods and containers, from the risk findings.
 
+<br />
+
+### Use Cloudanix with Kubernetes 1.25 and higher
+
+When you use the Cloudanix 1.0.x pack with Kubernetes 1.25 and higher, you need to add the **Spectro Namespace Labeler** add-on pack to your cluster profile. After you create the cluster profile, you then apply it to your cluster. 
+
+Use the following information to find the **Spectro Namespace Labeler** add-on pack.
+
+- **Pack Type**: System App
+- **Registry**: Public Repo
+- **Pack Name**: Spectro Namespace Labeler
+- **Pack Version**: 1.0.x or higher
+
+
+Below is the YAML file.
```

Comment:
Do we need to show this YAML? It could lead to confusion.

Reply:
@karl-cardenas-coding I'll ask @ajinkya1986